config/install/image.style.oe_theme_main_content_large.yml

```diff
@@ -0,0 +1,14 @@
+langcode: en
+status: true
+dependencies: {  }
+name: oe_theme_main_content_large
```

Comment:
Let's create image styles for the small breakpoints: 480 and 960 (1x and 2x), 768 and 1536 (1x and 2x). It's a bit bigger than the max size of the ecl-paragraph that wraps the news image but we can reuse them somewhere else.